### ISSUE TITLE
Additions

### ISSUE THREAD

**Issue by till** [itzg/kube-metrics-exporter#4]
Hey,

thanks again for publishing this.

Maybe you can reserve a port for the exporter and get yourself added to the official list:
- https://prometheus.io/docs/instrumenting/writing_exporters/#port-numbers
- https://github.com/prometheus/docs/blob/master/content/docs/instrumenting/exporters.md

LBNL, I am also looking if there is a way to assign labels from the namespace to the pod metrics. Would you take this as a contribution? Or is this too specific?

Thanks again,
Till

**Comment by itzg** [comment]
Great suggestion about the port assignment. I didn't even know that registry existed. 

I like the sound of your enhancement idea and would encourage a PR. From that I can better feel if it's too specific or not.

**Comment by itzg** [comment]
...oh, the dilemma of allocating a port is that the default is already 8080 (which is of course heavily over-used). As a user of this exporter, would you mind adjusting your prom config to a different port?

**Comment by till** [comment]
> ...oh, the dilemma of allocating a port is that the default is already 8080 (which is of course heavily over-used). As a user of this exporter, would you mind adjusting your prom config to a different port?

No, not at all. Just depends on how you interpret BC. No idea who uses this exporter, but it may require config changes for others.

**Comment by till** [comment]
On the additional labels: I think I gave up on the idea for now, since cadvisor would be more close to what I was trying to achieve.

**Comment by itzg** [comment]
Thanks for following up.
